app/views/forms/make_live/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.make_live_form'), @make_live_form.errors.any?)) %>
+<% content_for :back_link, govuk_back_link_to(form_path, "Back to create a form") %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @make_live_form, url: make_live_create_path) do |f| %>
+      <% if @make_live_form&.errors&.any? %>
+        <%= f.govuk_error_summary %>
+      <% end %>
+
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @make_live_form.form.name %></span>
+        Make your form live
+      </h1>
```

Comment:
Elsewhere we put the caption outside the heading, think we should probably do the same here, i.e.:
```suggestion
      <span class="govuk-caption-l"><%= @make_live_form.form.name %></span>
      <h1 class="govuk-heading-l">Make your form live</h1>
```

Reply:
At the moment the sub-heading is outside the h1 so for screenreaders they wouldn’t have context as to what the particular page relates too

> If the caption should be considered part of the page heading, you can also nest the caption within the <h1>.


https://design-system.service.gov.uk/styles/typography/#headings-with-captions

I had a chat with @thomasiles about this earlier in the week and we have updated the main form overview page already

Reply:
Makes sense!